src/components/MapView/index.tsx

```diff
@@ -17,10 +17,12 @@ const Map = ReactMapboxGl({
 function MapView({ classes }: MapViewProps) {
   const layers = useSelector(layersSelector);
   const serverAvailableDates = useSelector(availableDatesSelector);
-  const selectedLayersDates = serverAvailableDates.filter((dates, layerId) =>
-    layers.has(layerId),
+  const selectedLayersDates = serverAvailableDates.filter((_, layerId) =>
```

Comment:
For example in `src/config/layers.json`: 

```
"ndvi": {
    "title": "Modis NDVI",
    "server_type": "wms",
--> "server_layer": "ModisIndices",
    "server_uri": "http://185.30.8.6:5000/wms?styles=ndvi&layers=ModisIndices",
    "has_date": true,
    "date_interval": "month",
    "opacity": 0.3,
    "legend_text": "Modis NDVI"
},
```

The `layerId` is `ndvi` and `server_layer` is `ModisIndices`.

I will keep `layers.has(layerId)` 🤔?

Reply:
The names from the server and the layerId are not the same. We can discuss that if thtat's not clear but I will likely change a few things in #29